docs/cli-reference/gardenadm/gardenadm.md

```diff
@@ -16,7 +16,7 @@ gardenadm bootstraps and manages autonomous shoot clusters in the Gardener proje
 * [gardenadm connect](gardenadm_connect.md)	 - Deploy a gardenlet for further cluster management
 * [gardenadm discover](gardenadm_discover.md)	 - Conveniently download Gardener configuration resources from an existing garden cluster
 * [gardenadm init](gardenadm_init.md)	 - Bootstrap the first control plane node
-* [gardenadm join](gardenadm_join.md)	 - Bootstrap further control plane nodes or worker nodes and join them to the cluster
+* [gardenadm join](gardenadm_join.md)	 - Bootstrap worker nodes and join them to the cluster
```

Comment:
Can you extend the getting started docs?

(moved from https://github.com/gardener/gardener/pull/11934#discussion_r2073612294)

Reply:
Yes, I will, once #11934 is merged :)